packages/ragbits-core/src/ragbits/core/prompt/base.py

```diff
@@ -1,21 +1,47 @@
 from abc import ABCMeta, abstractmethod
-from typing import Dict, Generic, Optional, Type
+from typing import Any, Dict, Generic, Optional, Type
 
 from pydantic import BaseModel
 from typing_extensions import TypeVar
 
 ChatFormat = list[dict[str, str]]
+ChatFormatWithImages = tuple[ChatFormat, list[bytes]]
 OutputT = TypeVar("OutputT", default=str)
 
 
+class InputType(BaseModel):
```

Comment:
I don't understand why this Pydantic model is needed (and why it is public). Seems that that it holds static data (and not user-provided data that needs to be validated and/or serialized) that could be simple a static variable in this file.

Reply:
removed

Reply:
As of right now it seems to be still here

Reply:
oops. now its removed